File: manzerlacazFrontend/src/app/views/dashboard/Order.ts
export class Order{
  user_id:number;
  menu_id:number;
  payment:number;
  option:string;


  constructor(user_id: number, menu_id: number, payment: number, option: string) {
    this.user_id = user_id;
    this.menu_id = menu_id;
    this.payment = payment;
    this.option = option;
  }
}

File: manzerlacaz/manzerlacaz-entities/src/main/java/group/acensi/manzerlacaz/entities/Order.java
package group.acensi.manzerlacaz.entities;

import group.acensi.manzerlacaz.entities.base.AbstractEntity;
import jakarta.persistence.*;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Entity
@Data
@EqualsAndHashCode(callSuper = true)
public class Order extends AbstractEntity {
    
    private long user_id;
    private long menu_id;
	private long payment;
	private String option;

}
